website/models.py

```diff
@@ -63,6 +63,11 @@ class Company(models.Model):
     def __str__(self):
         return self.name
 
+    def save(self, *args, **kwargs):
+        if not self.company_id:
```

Comment:
Let's avoid the empty string case:
```suggestion
        if self.company_id is None:
```